### PR TITLE
defect in educationVo and inquiry education table resolved

### PR DESCRIPTION
**Pull request by jitendrajaria**

### CODE REVIEW

File: src/main/java/creativei/helper/ResponseHelper.java [hunk 1]
```diff
@@ -65,8 +65,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
     }
 
     private static List<EducationVo> getCreateEducationResponseData(List<InquiryEducation> inquiryEducation, List<EducationVo> educationVos) {
-        EducationVo educationVo=new EducationVo();
         for(InquiryEducation inquiryEducation1:inquiryEducation){
```
Comment (aayushKhandpur) [inline src/main/java/creativei/helper/ResponseHelper.java]: List name should be plural. Use proper naming conventions

File: src/main/java/creativei/helper/ResponseHelper.java [hunk 2]
```diff
@@ -65,8 +65,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
...
+            EducationVo educationVo=new EducationVo();
             educationVo.setId(inquiryEducation1.getId());
             educationVo.setAggregateMarks(inquiryEducation1.getAggregateMarks() == null ? null : inquiryEducation1.getAggregateMarks());
```
Comment (aayushKhandpur) [inline src/main/java/creativei/helper/ResponseHelper.java]: If we are setting null in case of null, this check can be avoided.

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address review comments

File: src/main/java/creativei/helper/ResponseHelper.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/helper/ResponseHelper.java]: issue solved in commit 6ffb6af

File: src/main/java/creativei/helper/ResponseHelper.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/helper/ResponseHelper.java]: okay resolved in commit 6ffb6af

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**